docs/source/containerization/kubernetes.mdx

```diff
@@ -64,10 +64,10 @@ kind: ServiceAccount
 metadata:
   name: release-name-router
   labels:
-    helm.sh/chart: router-1.0.0-rc.8
```

Comment:
It looks like this wasn't updated for the last release, so everything is "1" behind where it should be.  i.e:
1.5.0 should be 1.6.0
1.0.0-rc.9 should be 1.0.0-rc.10

And/Or: is some piece of automation being run too early?

Reply:
Yeah, this is it. The automated does regenerate stuff, but because we don't auto-increment before we do the helm update it's putting in the wrong thing.

Reply:
Generally this is fixed in [8816cbe](https://github.com/apollographql/router/pull/2260/commits/8816cbe3754974233580de3994268b08bd82c177) as I ran the release again, but then had to do a small cleanup in: [77179d7](https://github.com/apollographql/router/pull/2260/commits/77179d733e857c4c0e4dc8425d3af5b411ed490a)